config/services/twig.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+
+ This file is part of the Sylius package.
+
+ (c) Paweł Jędrzejewski
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+-->
+
+<container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <services>
+        <service id="sylius_price_history.twig.extension.sylius_version" class="Sylius\PriceHistoryPlugin\Application\Twig\SyliusVersionExtension">
```

Comment:
```suggestion
        <service id="Sylius\PriceHistoryPlugin\Application\Twig\SyliusVersionExtension">
```
according to the https://docs.sylius.com/en/1.12/book/contributing/code/conventions.html